components/infobox/wikis/formula1/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,114 @@
+---
+-- @Liquipedia
+-- wiki=formula1
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Role = require('Module:Role')
+local TeamHistoryAuto = require('Module:TeamHistoryAuto')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Player = Lua.import('Module:Infobox/Person', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+local Center = Widgets.Center
+
+local CustomPlayer = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _player
+
+---@param frame Frame
+---@return Html
+function CustomPlayer.run(frame)
+	local player = Player(frame)
+	_player = player
+
+	player.adjustLPDB = CustomPlayer.adjustLPDB
+	player.createWidgetInjector = CustomPlayer.createWidgetInjector
+
+	_args = player.args
+	_args.autoTeam = true
+
+	return player:createInfobox(frame)
+end
+
+---@param id string
+---@param widgets Widget[]
+---@rreturn Widget[]
+function CustomInjector:parse(id, widgets)
+	if id == 'history' then
+		local automatedHistory = TeamHistoryAuto._results{
+			convertrole = true,
+			player = _player.pagename,
+			addlpdbdata = true
+		}
+
+		if automatedHistory then
+			return {
+				Title{name = 'History'},
+				Center{content = {automatedHistory}},
+			}
+		end
```

Comment:
```suggestion
		return {
			Title{name = 'History'},
			Center{content = {TeamHistoryAuto._results{
				convertrole = true,
				addlpdbdata = true
			}}},
		}

```

Reply:
would add the history "header" even if no history is present